Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use an official Python runtime as a parent image
+FROM python:3.8-slim-buster
```

Comment:
I don't think we want to run it on 3.8 by default. Currently, we mainly use 3.11 and that's the version we should promote.